src/igvfd/schemas/changelogs/primary_cell.md

```diff
@@ -1,6 +1,11 @@
 ## Changelog for *`primary_cell.json'*
 
+### Schema version 7
```

Comment:
The primary_cell.json indicates version 6, but this changelog indicates version 7.